src/main/java/com/synopsys/integration/alert/provider/blackduck/BlackDuckProperties.java

```diff
@@ -114,11 +114,12 @@ public Integer getBlackDuckTimeout() {
         return Optional.empty();
     }
 
-    public HubServicesFactory createBlackDuckServicesFactory(final BlackduckRestConnection restConnection, final IntLogger logger) {
-        return new HubServicesFactory(HubServicesFactory.createDefaultGson(), HubServicesFactory.createDefaultJsonParser(), restConnection, logger);
+    public BlackDuckServicesFactory createBlackDuckServicesFactory(final BlackDuckRestConnection restConnection, final IntLogger logger) {
+        final Gson gson = BlackDuckServicesFactory.createDefaultGson();
```

Comment:
Should we create a defaultGson or autowire our version?

Reply:
We should update the Gson that we create as a Bean in Application if this is the Gson we want to use, and then autowire it in here to be used

Reply:
agreed we should autowire it in.